src/render/draw_data.rs

```diff
@@ -40,9 +41,9 @@ impl DrawData {
             }
         }
     }
-    pub(crate) unsafe fn cmd_lists(&self) -> &[*const DrawList] {
+    pub(crate) unsafe fn cmd_lists(&self) -> &[DrawList] {
```

Comment:
This returns a DrawList straight up, since DrawList now owns a pointer to ImDrawList, rather than being an ImDrawList itself